src/lbaf/IO/lbsGridStreamer.py

```diff
@@ -9,9 +9,8 @@ class GridStreamer:
     """A class containing to stream a grid with time-varying attributes
     """
 
-    def __init__(self, points, lines, field_arrays, point_arrays, cell_arrays, lgr: Logger = None):
-        """Class constructor
-        """
+    def __init__(self, points: vtk.vtkPoints, lines: vtk.vtkCellArray, field_arrays: dict = {}, point_arrays: list = [], cell_arrays: list = [], lgr: Logger = None):
```

Comment:
@ppebay line too long more than 120 chars
AND Default arguments value are mutable:
- field_arrays
- point_arrays
- cell_arrays

[Sample explanation](https://florimond.dev/en/posts/2018/08/python-mutable-defaults-are-the-source-of-all-evil/)

Reply:
BTW line 49 is too long as well

Reply:
That one was already there before that issue. Let's make it a separate issue -- I need to completely rework the grid streamer anyway.